recipes/counterfactual_data_augmentation.ipynb

```diff
@@ -27,20 +28,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Authentication\n",
     "import os\n",
     "\n",
-    "# !pip install kili # uncomment if you don't have kili installed already\n",
+    "!pip install --upgrade kili # uncomment if you don't have kili installed already\n",
     "from kili.client import Kili\n",
     "\n",
-    "api_endpoint = os.getenv('KILI_API_ENDPOINT') \n",
+    "# api_endpoint = os.getenv('KILI_API_ENDPOINT') \n",
     "# If you use Kili SaaS, use the url 'https://cloud.kili-technology.com/api/label/v2/graphql'\n",
+    "api_endpoint = \"https://cloud.kili-technology.com/api/label/v2/graphql\"\n",
     "\n",
-    "kili = Kili(api_endpoint=api_endpoint)\n",
+    "kili = Kili(api_endpoint=api_endpoint, api_key=\"6dc85cf3-3bbd-47db-bb26-1915f459489d\")\n",
```

Comment:
no private key in Github. If this is an active one you should revoke it.

Reply:
and it is assumed that the KILI_API_KEY is an environment variable

Reply:
I think the best way, given the default behavior of Kili client would be: 
```
# Don't forget to set your 'KILI_API_KEY' environment variable with your API Key: os.environ['KILI_API_KEY'] = \"<YOUR_API_KEY>\"
# If you use Kili on-premise, be sure to set your 'KILI_API_ENDPOINT' environment variable.
kili = Kili()
```

Reply:
Made the changes. Please have a look and resolve if poss.